backend/src/main/java/de/lindatroesken/backend/BackendApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
 
-@SpringBootApplication
+//@SpringBootApplication
+@SpringBootApplication(exclude = {SecurityAutoConfiguration.class })
```

Comment:
Wofür ist der exclude?

Reply:
Ohne security konnte ich mein Backend nicht starten. Ich weiß nicht an welcher Stelle, aber Spring integriert dann automatisch security, wenn ich es in dieser Zeile nicht explizit unterdrücke. Es läuft so und wird im nächsten Schritt eh ersetzt, durch meine eigene Config.